parser_test.go

```diff
@@ -69,9 +69,19 @@ func TestDefaultParser(t *testing.T) {
 
 	assert.Contains(t, whoisInfo.Registrar.DomainName, "GOOGLE.COM")
 	assert.Contains(t, whoisInfo.Registrar.DomainName, "google.com")
+	assert.Contains(t, whoisInfo.Registrar.RegistrarName, "MarkMonitor, Inc.")
 	assert.Equal(t, whoisInfo.Registrant.Country, "US")
 	assert.Equal(t, whoisInfo.Registrant.Province, "CA")
 	assert.Equal(t, whoisInfo.Registrant.Organization, "Google LLC")
+	assert.Equal(t, whoisInfo.Registrant.ID, "")
```

Comment:
If you want to add a test that checks a particular feature or proves a particular bug, please create a separate test called the same as the check you are doing in it. For example: `TestDefaultParserDoesNotCaptureEmptyGroups`

And create a separate fixture for it, not use existing one.

Reply:
Ok, now i`m moved this test to a new function and use run.com whois for a test data.